src/server/cluster/outgoing_slot_migration.h

```diff
@@ -35,6 +35,7 @@ class OutgoingMigration : private ProtocolClient {
   void Cancel(uint32_t shard_id);
 
   MigrationState GetState() const;
+  void Ack();
```

Comment:
Please add a comment what high level ack means because I see it calls Cancel on all flows